challenges/repeated-word/lib/repeatedWord.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (book) => {
+  if (typeof book !== 'string') {
+    throw new Error('Input Error: input must be a string');
+  }
+  // remove unnecessary punctuation. 
```

Comment:
Nice work commenting the regex. Regex is a perfect example of when to comment code, since it's not that human readable.